bundles/org.openhab.binding.samsungtv/src/main/java/org/openhab/binding/samsungtv/internal/discovery/SamsungTvDiscoveryParticipant.java

```diff
@@ -45,61 +48,47 @@ public class SamsungTvDiscoveryParticipant implements UpnpDiscoveryParticipant {
 
     @Override
     public Set<ThingTypeUID> getSupportedThingTypeUIDs() {
-        return Set.of(SAMSUNG_TV_THING_TYPE);
+        return Collections.singleton(SAMSUNG_TV_THING_TYPE);
```

Comment:
Why change this? As far as i know the singleton is only adding support for null and i don;t think this will ever be null.

Reply:
I didn't change this, as far as I can recall.

Reply:
This was changed by #15565, so it seems like you might have unintentionally at some point overwritten files committed to main while you were working on this PR?